notifier/notifier.go

```diff
@@ -383,25 +391,25 @@ func (n *Notifier) Stop() {
 
 // Describe implements prometheus.Collector.
 func (n *Notifier) Describe(ch chan<- *prometheus.Desc) {
```

Comment:
I filed an issue on being able to scope a registered with a label before passing it down, i.e. currying, some time ago.

It's a critical requirement for same non-globals.